src/JuliusSweetland.OptiKey/Services/WindowManipulationService.cs

```diff
@@ -415,6 +415,15 @@ public void Move(MoveToDirections direction, double? amountInPx)
             }
         }
 
+        public void TemporarilyResizeDockToSpecifiedHeight(double heightAsPercentScreen)
```

Comment:
Could this signature be "ResizeDockToSpecificHeight(double heightAsPercentScreen, bool persistNewSize)"